beacon-chain/blockchain/process_block.go

```diff
@@ -528,23 +527,6 @@ func (s *Service) handleEpochBoundary(ctx context.Context, postState state.Beaco
 	return nil
 }
 
-// This feeds in the block to fork choice store. It's allows fork choice store
-// to gain information on the most current chain.
-func (s *Service) insertBlockToForkchoiceStore(ctx context.Context, blk interfaces.ReadOnlyBeaconBlock, root [32]byte, st state.BeaconState) error {
-	ctx, span := trace.StartSpan(ctx, "blockChain.insertBlockToForkchoiceStore")
-	defer span.End()
-
-	if !s.cfg.ForkChoiceStore.HasNode(blk.ParentRoot()) {
```

Comment:
This check is performed at the beginning of `onBlock`

Reply:
It's different though, `onBlock` doesn't call `fillInForkChoiceMissingBlocks`. Only `onBlockBatch` does

Reply:
I didn't remove that helper

Reply:
My bad. I missed the "can't be triggered" part